### PR TITLE
Smart Info

### PR DESCRIPTION
**Pull request by MattToast**
Add a smart info target for info about current SS installation

Example Output:
```
$ smart info

Smart Python Packages:
╒════════════╤═══════════╕
│ Name       │ Version   │
╞════════════╪═══════════╡
│ SmartSim   │ 0.5.0     │
│ SmartRedis │ 0.4.1     │
╘════════════╧═══════════╛

Orchestrator Configuration:
╒═══════════╤════════════════════════════════════════════════════════════╕
│ Installed │ REDIS                                                      │
│ Location  │ /Users/mrdro/repos/ssimdev/ss/smartsim/_core/bin/redis-cli │
╘═══════════╧════════════════════════════════════════════════════════════╛

Redis AI Configuration:
╒══════════════╤═════════════════════════════════════════════════════════════╕
│ Is Installed │ Installed                                                   │
│ Location     │ /Users/mrdro/repos/ssimdev/ss/smartsim/_core/lib/redisai.so │
╘══════════════╧═════════════════════════════════════════════════════════════╛

Machine Learning Backends:
╒════════════╤═════════════════════╤══════════════════╕
│ Name       │ Backend Available   │ Python Package   │
╞════════════╪═════════════════════╪══════════════════╡
│ Tensorflow │ True                │ 2.8.0            │
│ Torch      │ True                │ 1.11.0           │
│ ONNX       │ False               │ Not Installed    │
╘════════════╧═════════════════════╧══════════════════╛

### CODE REVIEW

File: smartsim/_core/_cli/info.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    print("Orchestrator Configuration:")
+    db_path = _utils.get_db_path()
+    db_table = [["Installed", _fmt_installed_db(db_path)]]
+    if db_path:
+        db_table.append(["Location", str(db_path)])
+    print(tabulate(db_table, tablefmt="fancy_outline"), end="\n\n")
+
+    print("Redis AI Configuration:")
+    rai_path = _helpers.redis_install_base().parent / "redisai.so"
+    rai_table = [["Is Installed", _fmt_installed_redis_ai(rai_path)]]
```
Comment (al-rigazzi) [inline smartsim/_core/_cli/info.py:38]: I'm sure you thought about this for a lot (isn't this the kind of things we spend most of the dev time?) but I don't _love_ the redundancy in `Is installed | Installed` - what about one of the following two:
- `Is installed | Yes/No`
- `Status | Installed/Not installed`

And is there any chance we can also output the Redis AI version?

**Review by al-rigazzi — CHANGES_REQUESTED** [review]
Just one thing to think about and then looks great!

File: smartsim/_core/_cli/info.py (hunk 1 above)
Comment (MattToast) [reply smartsim/_core/_cli/info.py:38]: LOL, I also didn't love the redundancy! Changing to `Status`!

Comment (MattToast) [reply smartsim/_core/_cli/info.py]: I couldn't find an immediate way to get the RAI version without saving off some persistent artifact from the build process or without spinning up Redis Server, both of which seemed like a lot of work to try and and cram into a "quick and dirty" implementation for this release.

This is absolutely something that I was planning on tackling this task "properly" in a follow on PR next sprint!

**Review by al-rigazzi — APPROVED** [review]
LGTM!

**Comment by Spartee** [comment]
@MattToast @al-rigazzi This is sexy.

**Pull request merged**